app/assets/javascripts/shared/inline_editing.js

```diff
@@ -42,9 +45,17 @@ function initInlineEditing(title) {
         type: 'PUT',
         dataType: 'json',
         data: params,
-        success: function() {
+        success: function(result) {
+          var viewData;
+          if (editBlock.dataset.responseField) {
```

Comment:
In what circumstance does this `if` make sense. Basically we just ignore the successful update?

Reply:
By default we take string from input element, but if we want to modify this string on backend, we need to provide this data field.
In our case case in input string we must save raw text, but in preview show with html tags.